eos/models/wilson-coefficients.hh

```diff
@@ -168,23 +168,37 @@ namespace eos
          * These two operators are defined via
          *
          * H^eff = 4 G_F / sqrt(2) V_tb V_ts^* (
-         *             C_L alpha_e / (2 pi) * [sbar gamma^mu P_L b] [nubar gamma_mu P_L nu]
-         *             C_R alpha_e / (2 pi) * [sbar gamma^mu P_R b] [nubar gamma_mu P_L nu]
+         *             C_L alpha_e / (2 pi) * [nubar gamma_mu P_L nu] [sbar gamma^mu P_L b] 
+         *             C_R alpha_e / (2 pi) * [nubar gamma_mu P_L nu] [sbar gamma^mu P_R b] 
          *         )
-         *       = 4 G_F / sqrt(2) V_tb V_ts^* ( C_L O_L + C_R O_R )
+         *       = 4 G_F / sqrt(2) V_tb V_ts^* ( C_VLL O_VLL + C_VLR O_VLR )
+         * 
+         * Furthermore, we can add the scalar modes [FLS:2021A], eq. (2)
+         * O_SLL = [bar(nu_L^c) P_L nu][sbar P_L b], where nu_L^c = C bar(nu_L)^T, C = i gamma^2 gamma^0
+         * O_SLR = [bar(nu_L^c) P_L nu][sbar P_R b], where nu_L^c = C bar(nu_L)^T, C = i gamma^2 gamma^0
+         * 
+         * And tensor modes [FLS:2021A], eq. (2)
+         * O_TLL = [bar(nu_L^c) sigma^{mu nu} P_L nu][sbar sigma_{mu nu} P_L b], where nu_L^c = C bar(nu_L)^T, C = i gamma^2 gamma^0
+         * 
+         * 
          *
          * C++ idx -> operator
          * 0       -> O_L
```

Comment:
Please adjust the notation of the vector operators to reflect your renaming in the code, i.e., use ``VLL` and ``VRL``.